README.md

```diff
@@ -154,12 +181,6 @@ Edit the `ios/YourProjectName/AppDelegate.m` file:
 }
 ```
 
-Set the `BootSplash.storyboard` as launch screen file:
```

Comment:
Setting the storyboard automatically in Xcode **without any user consent** is not a good idea IMHO.
In our project, we have 3 different `Info.plist` with 3 different splash screens, and I don't want the CLI to try doing magic for me in this particular case (we have a `BootSplashDevelopment.storyboard`, `BootSplashStaging.storyboard` and `BootSplashProduction.storyboard`, I don't want to edit a single plist automatically).

But the feature is dope! So I think it will be great to keep it, add a `yes / no` confirm step (`yes` will be the default) and a CLI flag to disable it (like `--no-plist-setup` or any better name 😅) and set this part of the documentation as optional (_if you choose not to setup automatically…_).

Reply:
Perfectly fair point. The addToXcode is now a separate confirmation in `yarn generate-bootsplash` and a separate flag in `react-native generate-bootsplash`. (Also deployed as `react-native add-bootsplash-to-xcode`

Reply:
It works as you hope: running `react-native generate-bootsplash` introduces the same workflow as you had previously with the additional option of adding the bootsplash to the pbxproj and plist files.

I updated the README in the PR to make this all clearer.